mapie/utils.py

```diff
@@ -412,6 +412,29 @@ def check_gamma(
         )
 
 
+def get_effective_calibration_samples(scores: NDArray, sym: bool):
```

Comment:
Is the `sym` not by default `True`?

Reply:
The `sym` is by default `True` in `ConformityScore`. In any case, we need to compute the effective calibration sample size to compute the quantile.